src/xmipp/SConscript

```diff
@@ -383,10 +383,9 @@ def addMatlabBinding(name):
     matlabDir = join(XMIPP_PATH, 'bindings', 'matlab')
     source = join(matlabDir, name)
     target = join(matlabDir, name.replace(".cpp",".mexa64"))
-
-    cmdTarget = env.Command(target, source, compileMatlabBinding, deps=XMIPP_LIBS)
+    env.Depends(target = target, dependency = XMIPP_LIBS)
```

Comment:
what is the purpose of this dependency?

Reply:
To force Scons to compile the Matlab binding once the Xmipp libraries are compiled.